documentation/contributing/pullrequests.markdown

```diff
@@ -0,0 +1,85 @@
+This makes it much more likely to get your changes merged soon, because if maintainers are unsure about one change, they can still merge the other one after review.
+
+### ping
+
+If you feel everyone forgot about your request, add a short note saying "ping" or "Could someone please review/merge this?" to the request. It might be that the item dropped of everyone's TODO list for no reason.
+
+### review requests and +1
+
+All feedback on existing pull-requests is welcome and appreciated. If you are familiar with the code someone requests to change, please step up and review their requests. The maintainer's time is often quite limited. This usually boils down to answering the following questions and provide feedback in the request:
+
+ - Does the patch do what the author claims it does?
+ - Is it general enough to work with everyones use-case and not only the requestor's setup?
+ - Does it address all edge-cases you are aware of?
+ - If a new interface is introduced: Is this interface general enough for likely future modifications? Otherwise we might have to break API later on if we wish to add more useful features to the same module.
+ - Is the change [ABI compatible](https://en.wikipedia.org/wiki/Application_binary_interface) or is there a good reason (e.g. safety) to break it?
+ - Does the patch respect the [MoveIt! Code Style](code.html)?
+
+If (and only if) you answer all these questions with "yes", add a note saying "+1" to the request. This does **not** mean you like to have the claimed feature in MoveIt!. If it's not the first message you ever sent on a public channel concerning MoveIt, it means the next maintainer who reviews this request is free to merge it after review.
+If you are unsure about some part of the request, feel free to add a note and ask the author for details and explanations of individual changes.
+
+## Maintainer Side
+
+The ``*-devel`` branches of the repositories must always compile on the target platforms.
+
+The ``*-devel`` branches must not be force-pushed.
+
+### no direct commits
+
+No one is allowed to directly commit to the ``*-devel`` branches of the repositories. Instead they should create feature branches and add pull-requests.
+The only commits that might be pushed directly are cherry-picks from older branches (see below) and administrative changes (e.g. CHANGELOGS, tags).
+
+### 1+1 == merge
+
+If you see a trustworthy "+1" note in a request or the requestor is a fellow maintainer and all feedback has been addressed by the requestor,
+merge the request after your own review. Otherwise add a "+1" note after you are satisfied after the review.
+
+### squash-merge if possible
+
+Pull-requests with a single commit should *always* be squash-merged (github supports this via a drop-down list on the "merge" button).
```

Comment:
Question for my ignorance: if a PR has only 1 commit, is there a difference between a squash-merge and a regular merge? Won't both result in a single commit with the same title?